pkg/rpmdb_testcase.go

```diff
@@ -2792,6 +2792,194 @@ var (
 		{intRef(), "ostree-libs", "2019.6", "2.el8", "x86_64", "ostree-2019.6-2.el8.src.rpm", 981292, "LGPLv2+", "CentOS", "", "Development headers for ostree", "RSA/SHA256, Thu Apr  9 03:59:21 2020, Key ID 05b555b38483c65d", PGPHASHALGO_SHA256, nil, nil, nil, nil, nil, nil, nil, nil, nil},
 	}
 
+	// docker run --rm -it centos:8 bash
```

Comment:
This might be more repeatable test if we used
`docker pull --platfrom linux/s390x centos:streams9`

Reply:
Updated.